### PR TITLE
Check examples with http protocol instead of file

### LINKED ISSUE #5712: Fetch (polyfill?) no longer works for file:// urls in PhantomJS
Travis is failing with the following stack trace for examples that use `fetch()`:
```
JavaScript ERROR: Uncaught (in promise) file:///Users/ahocevar/projects/ol3/build/hosted/master/examples/flight-animation.js:72:29
callHanler@https://cdn.polyfill.io/v2/polyfill.js?features=fetch,requestAnimationFrame,Element.prototype.classList,URL:1656:24
tryCatcher@https://cdn.polyfill.io/v2/polyfill.js?features=fetch,requestAnimationFrame,Element.prototype.classList,URL:1373:38
https://cdn.polyfill.io/v2/polyfill.js?features=fetch,requestAnimationFrame,Element.prototype.classList,URL:1512:39
flush@https://cdn.polyfill.io/v2/polyfill.js?features=fetch,requestAnimationFrame,Element.prototype.classList,URL:1415:20
```
It turns out that network requests through file urls are not handled by the polyfill.

### PR DESCRIPTION
**Pull request by ahocevar**
Fixes #5712.

### CODE REVIEW

**Comment by ahocevar** [comment]
When merged, Travis will give green for master again.

**Comment by tschaub** [comment]
Nice work @ahocevar.  Please merge.

**Comment by tsauerwein** [comment]
The latest build failed on master:

```
./node_modules/.bin/serve-files 8000 . & pid=$! && ./node_modules/.bin/phantomjs --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js http://localhost:8000/build/hosted/HEAD/examples/reprojection-by-code.html && kill -9 $pid

PAGE LOAD ERROR

  phantomjs://code/check-example.js:27

Server is serving files from . waiting for connections on http://localhost:8000/ - have fun!

make: *** [build/timestamps/check-reprojection-by-code-timestamp] Error 1
```
https://travis-ci.org/openlayers/ol3/builds/151729122

Could it be that sometimes the file server has not yet started? Should we wait until `localhost:8000` responds?

File: Makefile [hunk 1]
```diff
@@ -184,7 +184,7 @@ build/timestamps/check-%-timestamp: $(BUILD_HOSTED)/examples/%.html \
                                     $(BUILD_HOSTED)/build/ol.js \
                                     $(BUILD_HOSTED)/css/ol.css
 	@mkdir -p $(@D)
-	./node_modules/.bin/phantomjs --local-to-remote-url-access=true --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js $<
+	./node_modules/.bin/serve-files 8000 . & pid=$$! && ./node_modules/.bin/phantomjs --ssl-protocol=any --ignore-ssl-errors=true bin/check-example.js http://localhost:8000/$< && kill -9 $$pid
```
Comment (gberaudo) [inline Makefile]: Shouldn't it be `;` instead of `&&` before the `kill` command?

**Comment by ahocevar** [comment]
Follow-up pull request created: #5715.

**Pull request merged**